### PR TITLE
Noise repellent : 0.1.5 -> 0.2.1

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/noise-repellent/default.nix [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+  version = "0.2.1";
 
   src = fetchFromGitHub {
     owner = "lucianodato";
     repo = pname;
-    rev = version;
-    sha256 = "0hb89x9i2knzan46q4nwscf5zmnb2nwf4w13xl2c0y1mx1ls1mwl";
+    rev = "v${version}";
+    sha256 = "sha256-hMNVzhJZFGFeu5aygLkfq495O0zpaIk41ddzejvDITE=";
     fetchSubmodules = true;
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/noise-repellent/default.nix:12]:
```suggestion
```
has not submodules

File: pkgs/development/libraries/audio/libspecbleach/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  buildInputs = [
+    fftwFloat
+  ];
+
+  meta = with lib; {
+    description = "C library for audio noise reduction";
+    homepage    = "https://github.com/lucianodato/libspecbleach";
+    license     = licenses.lgpl2;
+    maintainers = [ maintainers.magnetophon ];
+    platforms = [ "x86_64-linux" "i686-linux" "x86_64-darwin" "i686-darwin"  ];
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/audio/libspecbleach/default.nix:24]: Does that really not support aarch64?

```suggestion
    platforms = [ "x86_64-linux" "i686-linux" "x86_64-darwin" "i686-darwin" ];
```

Comment (magnetophon) [reply pkgs/development/libraries/audio/libspecbleach/default.nix:24]: Seems it does: https://github.com/lucianodato/noise-repellent/commit/508fc294fb79d51779339452fd579a48059cde9b.

Comment (SuperSandro2000) [reply pkgs/development/libraries/audio/libspecbleach/default.nix:24]: So can we also change that to unix?

Comment (magnetophon) [reply pkgs/development/libraries/audio/libspecbleach/default.nix]: Done.

**Pull request merged**